### DISCUSSION TITLE
Watchtower permission denied error after docker update

### DISCUSSION THREAD

**Discussion by Nabukodonosor** [containrrr/watchtower#1974] [Q&A]
So, I'm using watchtower in docker and Openmediavault 7. So, I was doing some stuff on my NAS, and I saw a few updates in the OMV7 dash, so I thought let me update this and go on with my day. I did the update, these 3 below:
https://prnt.sc/ZJG50tBR3XZR
After this, I started getting tons of emails from watchtower, all like this:
`permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.25/containers/json?filters=%7B%22status%22%3A%7B%22running%22%3Atrue%7D%7D": dial unix /var/run/docker.sock: socket: permission denied`

I had to stop it in CLI because I was getting tons of them.

I couldn't start any of my dockers, still can't. Except of maybe 3 of them. OMV can start, Flame, and notifiarr.

Tried restarting docker, server, nothing worked.

My docker compose:

`---
services:
  watchtower:
    image: containrrr/watchtower
    container_name: watchtower
    environment:
      - PUID=1001 #dockeruser, added to Docker group
      - PGID=100
      - TZ=Europe/Belgrade
      - WATCHTOWER_CLEANUP=true
      - WATCHTOWER_MONITOR_ONLY=false
      - WATCHTOWER_INCLUDE_STOPPED=Europe/Belgrade
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
    restart: unless-stopped`

Downgrading Docker to previous version 26.0.1 didn't help.

Any ideas?

**Comment by Nabukodonosor** [comment]
Solved by:
sudo apt-get purge apparmor

**Reply by Macrisu** [reply]
`WATCHTOWER_INCLUDE_STOPPED=Europe/Belgrade. 
`
should be

`WATCHTOWER_INCLUDE_STOPPED=true`

**Reply by Nabukodonosor** [reply]
That was a typo, it says "true".
